layers/landcover/style.json

```diff
@@ -18,19 +18,7 @@
       },
       "paint": {
         "fill-color": "#fff",
-        "fill-opacity": {
-          "base": 1,
-          "stops": [
-            [
-              0,
-              0.9
-            ],
-            [
-              10,
-              0.3
-            ]
-          ]
-        }
+        "fill-opacity": 0.9
```

Comment:
Do we want all styles to be fully opaque (e.g. 1)? Probably it makes little difference either way. 🤷